### DISCUSSION TITLE
windows wezterm image

### DISCUSSION THREAD

**Discussion by zaeboba** [fastfetch-cli/fastfetch#1339] [Q&A] [answered]
how to display image in wezterm on windows? i already installed viu (yes i used gpt chat) but nothing works. 
![изображение](https://github.com/user-attachments/assets/15fd36ce-4a18-45cd-a530-fcf1dc16812a)

**Comment by Neo3477** [comment]
Can you share your config? This is how it looks for me with Wezterm, though on macOS.


![E702AEA0-1996-4F7C-8CC2-49EAD18DD109](https://github.com/user-attachments/assets/145678e3-9c86-4f71-9c1e-7be7d478cef9)




`{
    "$schema": "https://github.com/fastfetch-cli/fastfetch/raw/dev/doc/json_schema.json",
    
    "logo": {
         "type": "iterm",
         "source": "~/.config/fastfetch/sans.png",
         "width": 40,
         "height": 18
     },
    
        "modules": [
        "break",
        {
            "type": "custom",
            "format": "┌────────────────────── Hardware ──────────────────────┐",
            "outputColor": "red"
        },
        {
            "type": "title",
            "key": " PC",
            "keyColor": "green"
        },
        {
            "type": "cpu",
            "key": "│ ├󰍛CPU",
            "showPeCoreCount": true,
            "format": "{1}",
            "keyColor": "green"
        },
        {
            "type": "gpu",
            "key": "│ ├󰍛 GPU",
            "keyColor": "green"
        },
        {
            "type": "memory",
            "key": "└ └󰍛 Memory",
            "keyColor": "green"
        },
        {
            "type": "custom",
            "format": "└──────────────────────────────────────────────────────┘",
            "outputColor": "red"
        },
        "break",
        {
            "type": "custom",
            "format": "┌────────────────────── Software ──────────────────────┐",
            "outputColor": "red"
        },
        {
            "type": "os",
            "key": " OS",
            "keyColor": "yellow"
        },
        {
            "type": "kernel",
            "key": "│ ├ Kernel",
            "keyColor": "yellow"
        },
        {
            "type": "packages",
            "key": "│ ├󰏖 Packages",
            "keyColor": "yellow"
        },
        {
            "type": "shell",
            "key": "│ ├ Shell",
            "keyColor": "yellow"
        },
        {
            "type": "command",
            "key": "│ ├ OS Age",
            "keyColor": "yellow",
            "text": "birth_install=$(stat -c %W /); current=$(date +%s); time_progression=$((current - birth_install)); days_difference=$((time_progression / 86400)); echo $days_difference days"
        },
        {
            "type": "uptime",
            "key": "└ └ Uptime",
            "keyColor": "yellow"
        },        
        {
            "type": "custom",
            "format": "└──────────────────────────────────────────────────────┘",
            "outputColor": "red"
        },
        {
            "type": "custom",
            "format": "               \u001b[90m  \u001b[31m  \u001b[32m  \u001b[33m  \u001b[34m  \u001b[35m  \u001b[36m  \u001b[37m "
        },
        "break",
    ]
}
`

**Reply by FanaticExplorer** [reply]
Terribly sorry for off top, but can you share that sans image? It looks neat!

**Comment by zaeboba** [comment]
```
{
    "$schema": "https://github.com/fastfetch-cli/fastfetch/raw/dev/doc/json_schema.json",
    "logo": {
     "type": "kitty",
     "source": "C:\\Users\\jacka\\.config\\fastfetch\\IMG_1329.png",
     "width": 40,
     "height": 18
 },
```
 or
```
 {
    "$schema": "https://github.com/fastfetch-cli/fastfetch/raw/dev/doc/json_schema.json",
    "logo": {
        "type": "file",
        "source": "C:\\Users\\jacka\\.config\\fastfetch\\IMG_1329.png",
        "color": {
            "1": "cyan",
            "2": "red",
            "3": "yellow",
            "4": "blue",
            "5": "green"
        }
    },
```

**Reply by CarterLi** [reply, marked as answer]
You should use `"type": "iterm"`

**Reply by zaeboba** [reply]
Thank you very much! that helped! i'm not a programmer and i'm not very good with documentation)
![изображение](https://github.com/user-attachments/assets/c05426ce-1d85-4ec9-b0cb-6eb87eb0dba4)
only there was some kind of font error, but that's going to be off-topic.

**Comment by Neo3477** [comment]
I think you have to write the path differently, have a look here under point 1.:

https://github.com/fastfetch-cli/fastfetch/wiki/Logo-options

"%USERPROFILE%\\.config\fastfetch\IMG_1329.png"

**Reply by zaeboba** [reply]
![изображение](https://github.com/user-attachments/assets/d38e35a0-08ff-4198-bdcb-99fafac08307)
The first one is with a “\” and the second one is with a “\\\”
